nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
> (Sorry, to be more clear in my comment, why is there a menu_margin at all? Do other browsers do that? I don't see that effect in either Chrome or Firefox on my computer....)

I have this effect on my browser. It is more of a buffer so that the menu doesn't press right up against the edge of the window even if it would have fitted in.